iothub/device/tests/Transport/Http/HttpTransportHandlerTests.cs

```diff
@@ -91,7 +91,12 @@ public void HttpTransportHandler_DisableReceiveMessageAsync_ThrowsNotSupportedEx
 
         HttpTransportHandler CreateFromConnectionString()
         {
-            return new HttpTransportHandler(new PipelineContext(), IotHubConnectionInfoExtensions.Parse(DumpyConnectionString), new IotHubClientHttpSettings());
+            return new HttpTransportHandler(
+                new PipelineContext
+                {
+                    IotHubConnectionInfo = new IotHubConnectionInfo(new IotHubConnectionStringBuilder(DumpyConnectionString), new IotHubClientOptions(new IotHubClientHttpSettings())),
```

Comment:
Haha dumpy?

Reply:
iirc we used to call these "dummy" but the code analysis tools kept complaining.

Reply:
The word you're looking for is `Fake`.